test/repositoryUtils.test.js

```diff
@@ -0,0 +1,62 @@
+import {expect} from 'chai';
+import {describe , it} from 'mocha';
+
+import { filterRepositoriesWithMoreThanFiveStars, getLastFiveUpdatedRepositories, sumOfAllRepositoryStars }  from '../src/repositoryUtils.js';
+import { mockedRepositories, mockedRepositoriesFull, mockedRepositoriesLess }  from './mockedData.js';
+
+describe('filterRepositoriesWithMoreThanFiveStars', () => {
+    describe('when it receives an array of objects', () => {
+        it('should return repositories with more than 5 stars', () => {
```

Comment:
let's avoid using should in the tests.
```suggestion
        it('returns repositories with more than 5 stars', () => {
```

Reply:
Should  erased from the tests